trustpoint/pki/views/domains.py

```diff
@@ -104,11 +106,25 @@ class DomainConfigView(DomainContextMixin, DomainDevIdRegistrationTableMixin, Li
     detail_context_object_name = 'domain'
     success_url = reverse_lazy('pki:domains')
 
-    def get_context_data(self, **kwargs):
+    def get_context_data(self, **kwargs: Any) -> dict[str, Any]:
         """Adds (no) additional context data."""
         context = super().get_context_data(**kwargs)
         domain = self.get_object()
 
+        issued_credentials = domain.issued_credentials.all()
+
+        certificates = CertificateModel.objects.filter(
+            credential__in=[issued_credential.credential for issued_credential in issued_credentials]
+        )
+
+        context['certificates'] = certificates
+        context['protocols'] = {
```

Comment:
Is this code with the protocols (scep, acme, ...) in domains new code or a regression? I believe I recall we removed it a while back...